### PR TITLE
build: Adjust package dependencies to compile in Fedora

### PR DESCRIPTION
**Pull request by ueno**
This makes it possible to build this crate with the Rust toolchain in
Fedora, so the keylime_agent binary can be packaged as an RPM.

The commit comprises the following:
- Use tss-esapi 6.1.1 from crates.io
- Bump rust-ini dependency to 0.17
- Bump pretty_env_logger dependency to 0.4
- Bump hex dependency to 0.4
- Move wiremock to dev-dependencies

Signed-off-by: Daiki Ueno <dueno@redhat.com>

Fixes: #239

### CODE REVIEW

**Comment by ueno** [comment]
Preliminary packages are available at https://copr.fedorainfracloud.org/coprs/ueno/rust-keylime/

File: src/common.rs [hunk 1]
```diff
@@ -230,7 +230,7 @@ pub(crate) fn config_get(section: &str, key: &str) -> Result<String> {
         }
     };
 
-    Ok(value.clone())
+    Ok(value.to_string())
```
Comment (lukehinds) [inline src/common.rs:233]: looks like this might be a separate matter? (or is this related to the dependency bump?)

Comment (ueno) [reply src/common.rs:233]: This is needed for rust-ini bump from 1.12.1 to 1.17.0. In 1.12.1, `value` is supposed to be a `String`, because `ini::ini::Properties` is an alias to [`HashMap<String,String>`](https://docs.rs/rust-ini/0.12.1/ini/ini/type.Properties.html), while in 1.17.0, it is redefined as [a dedicated struct](https://docs.rs/rust-ini/0.17.0/ini/struct.Properties.html#method.get), whose `get` method returns `&str`; so a manual conversion to `String` is necessary.

Comment (lukehinds) [reply src/common.rs:233]: makes sense, thanks!

**Comment by ueno** [comment]
I've also updated the `tss-esapi` dependency  to 6.0 as it includes @puiterwijk's patch.

**Comment by lukehinds** [comment]
LGTM @ueno , if you could rebase this I am happy to approve.

**Comment by ueno** [comment]
Thanks; rebased and removed the `tss-esapi` update (as it's already part of https://github.com/keylime/rust-keylime/commit/31ac452fb42dbb0654903eb1a72c17c8c6173200).

**Review by lukehinds — APPROVED** [review]

**Comment by ueno** [comment]
@lkatalin @lukehinds now that tss-esapi 6.1.1 has been published on crates.io, I've incorporated the changes needed for using that version (plus removal of `unsafe` block which is no longer necessary, and `Tcti` -> `TctiNameConf` [transition](https://github.com/parallaxsecond/rust-tss-esapi/blob/6.x.y/tss-esapi/src/lib.rs#L102)).

**Comment by lkatalin** [comment]
@ueno This looks good to me, awesome work, thank you!
I guess the changes I pinned to earlier from the [tss-esapi github](https://github.com/parallaxsecond/rust-tss-esapi) (ex. `SignatureSchemeAlgorithm` instead of `SignatureScheme`) are not published yet in `6.1.1` so we have to revert them for now.

**Review by lkatalin — APPROVED** [review]

**Comment by ueno** [comment]
Thanks for the review!
